.vale/fixtures/RedHat/Definitions/testvalid.adoc

```diff
@@ -1,6 +1,8 @@
+// suppress inspection "IncorrectFormatting" for whole file
```

Comment:
What does this mean?

Reply:
That's a comment that IntelliJ added to suppress one inspection and disable autoformatting.